runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -388,6 +427,18 @@ public void onGameTick(GameTick event)
 		lastPlayerLocation = client.getLocalPlayer().getWorldLocation();
 	}
 
+	void updateList(String item)
+	{
+		final List<String> highlightedNpcs = new ArrayList<>(highlights);
+
+		if (!highlightedNpcs.removeIf(item::equalsIgnoreCase))
+		{
+			highlightedNpcs.add(item);
+		}
+
+		config.setNpcToHighlight(Text.toCSV(highlightedNpcs));
```

Comment:
This works due to this triggering the config change event to rebuild npcs right? Would add
```java
// this triggers the config change event and rebuilds npcs
```